File: gateOne/day 2/java/Task2.java
import java.util.Arrays;
import java.util.Scanner;
public class Task2{
	public static void main(String... args){
		Scanner input = new Scanner(System.in);
		int[] scoreArray = new int[10];
		
		
		for(int count = 0; count < scoreArray.length; count++){
			System.out.println("enter score");
			int score = input.nextInt();
			scoreArray[count] = score;

		}
		
		String arraySave = new String[1];
		for(int count = 0; count < scoreArray.length; count++){
			arraySave[0] = scoreArray[count];
			String result = Arrays.toString(arraySave);
			
			System.out.println(result);

		}
		
		
		

	}
}

File: gateOne/day-Two/javascript/task2.js
let scoreArray = [];
const prompt = require("prompt-sync")();
for(var count = 0; count < 10; count++){
	var score = prompt("Enter score: ");
	scoreArray[count] = score;

}

for(var count = 0; count < scoreArray.length; count++){
	console.log(scoreArray[count]);
}
